src/backend/distributed/commands/transmit.c

```diff
@@ -31,12 +31,12 @@ static bool ReceiveCopyData(StringInfo copyData);
 
 
 /*
- * ReceiveRegularFile receives data from stdin using the standard copy
+ * RedirectTransmittedDataToRegularFile receives data from stdin using the standard copy
  * protocol. The function then creates or truncates a file with the given
  * filename, and appends received data to this file.
  */
 void
-ReceiveRegularFile(const char *filename)
+RedirectTransmittedDataToRegularFile(const char *filename)
```

Comment:
Are ReceiveRegularFile() and RedirectTransmittedDataToRegularFile() same?

Instead of copying ReceiveRegularFile to another function, can we use the same function in all places?